src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -24,11 +24,10 @@ private static unsafe void FillStringChecked(string dest, int destPos, string sr
                 throw new IndexOutOfRangeException();
             }
 
-            fixed (char* pDest = &dest._firstChar)
-            fixed (char* pSrc = &src._firstChar)
-            {
-                wstrcpy(pDest + destPos, pSrc, src.Length);
-            }
+            Buffer.Memmove(
+                destination: ref Unsafe.Add(ref dest._firstChar, destPos),
```

Comment:
The order of the arguments isn't the same for each of these calls. Is there a specific reason they are in a different order? Perhaps to demonstrate some priority of the arguments in different cases?

Reply:
The values are generally ordered the same as the arguments in the method signature. There are some exceptions where I moved the length count first, and those are called out in the comments.

I know I always have trouble remembering the order of arguments to methods like _memmove_, so the named parameters are an attempt to make the code a bit more readable. But admittedly it's not a practice we follow commonly throughout corelib. If there's value in having consistency with the rest of the code base I can remove the named arguments.

Reply:
Oh gotcha. That is the derefing comment. Makes sense now. Thanks.